doc/classes/ARVRController.xml

```diff
@@ -77,6 +78,7 @@
 	</methods>
 	<members>
 		<member name="controller_id" type="int" setter="set_controller_id" getter="get_controller_id">
+			The controller's id. The first controller that the ARVR Server detects will have id 1, the second id 2, the third id 3, etc. When a controller is turned off that slot is freed ensuring that controllers will keep the same id while it is turned on even when controllers with lower ids are turned off.
```

Comment:
- Some tense problems here:
> When a controller is turned off that slot is freed ensuring that controllers will keep the same id while it is turned on even when controllers with lower ids are turned off.

When a controller is turned off, its slot is freed. This ensures controllers will keep the same id even when controllers with lower ids are turned off.